docs/computing/containers/creating.md

```diff
@@ -42,3 +42,37 @@ Also see our [documentation on how to run containers](run-existing.md).
 You can also build your own container from scratch. This is an option for more experienced users, and your main source of information is the [official Apptainer documentation on building containers](https://apptainer.org/docs/user/main/build_a_container.html).
 
 You can find some help also by looking at our [tutorial on building Apptainer containers from scratch](../../support/tutorials/singularity-scratch.md).
+
+## Building a container without sudo access
+
+Root access into the cluster is not permitted. Namespaces have also been disabled due to security issues involved. However with a few restrictions Apptainer can still be used by an unprivileged user to build a container by using [fakeroot](https://apptainer.org/docs/user/main/fakeroot.html).
```

Comment:
Maybe it is just my personal preference, but I'd propose talking about "CSC supercomputers" instead of "cluster". Or if a specific system is referred to, then "Puhti", "Mahti", or "LUMI", for example.